utils/certs/gen_certificates.sh

```diff
@@ -22,7 +22,7 @@ CA_HOME="${1:-.}/daosCA"
 PRIVATE="${CA_HOME}/private"
 CERTS="${CA_HOME}/certs"
 CLIENTS="${CERTS}/clients"
-CONFIGS="$(dirname "${BASH_SOURCE[0]%/*}")"
+CONFIGS="$(dirname "${BASH_SOURCE}")"
```

Comment:
This only needed reverting because my [original suggestion](https://github.com/daos-stack/daos/pull/13360/files/88eb485b65e6a41053c0e94bfe18d7396d89d7b4#r1399504327) was incorrect but I [did correct it subsequently](https://github.com/daos-stack/daos/pull/13360/files/88eb485b65e6a41053c0e94bfe18d7396d89d7b4#r1406477480) but that suggestion was never implemented.